mdet_tests/test_mdet_regression.py

```diff
@@ -230,12 +230,19 @@ def test_mdet_regression(fname, write=False):
     else:
         old_data = fitsio.read(fname)
         for col in old_data.dtype.names:
+            if ('psf_' in col or 'ratio' in col) and '1.3.8' in fname:
+                rtol = 1.0e-4
+                atol = 1.0e-4
+            else:
+                rtol = 1.0e-5
+                atol = 2e-6
+
```

Comment:
We should make this change in precision conditional on whether or not we are looking at the code on master vs ngmix v1.

Reply:
v1 will not change right?  We already know it passes?

Reply:
We test the current code in the head of the PR against master and against v1.3.8. We should demand a better tolerance for the test against master than for that against v1.3.8.

Reply:
How can one test they are in master?  Or do you mean test if version >= 2?

Reply:
oh, sorry I misunderstood, you mean if 'master' is in the filename

Reply:
yeah - I checkout master and produce a benchmark. I also install 1.3.8 and produce a benchmark. Then I compare those benchmarks to what the code in the PR produces.

Reply:
it already only uses the low tolerance for '1.3.8' in the filename

Reply:
maybe I misunderstand?

Reply:
This was my mistake. Resolving and looks good!